cloudProviderData/main.go

```diff
@@ -0,0 +1,56 @@
+package main
```

Comment:
Go does not use camelCase for anything,.. ie. `cloudProviderData` is a no-go.

instead, the convention should be, `cmd/cloud-provider-data/main.go`

Reply:
done, my bad :+1: